src/lambda/txs.ts

```diff
@@ -21,12 +23,15 @@ const lambda = async (apiKey) => {
     }
   } catch (error) {
     console.error(error)
-    return { statusCode: 500, body: JSON.stringify({ msg: error.message }) }
+    return {
+      statusCode: 500,
+      body: JSON.stringify({ msg: (error as Error).message }),
+    }
   }
 }
 
 const handler = () => {
   return lambda(process.env.ETHERSCAN_API_KEY)
 }
 
-module.exports = { handler, lambda }
+export { handler, lambda }
```

Comment:
Would it be better to use named exports directly on the functions here now that we've dropped CommonJS?

Reply:
I agree, I was following this example: https://github.com/ethereum/ethereum-org-website/blob/dev/src/lambda/coinmetrics.ts

Reply:
Uhmm I missed that... @WilliamWelsh if you are down, I encourage you to switch them to named exports. Would be nice to have 👍🏼